zapisy/apps/theses/forms.py

```diff
@@ -82,6 +84,10 @@ def clean(self):
         if ('students' in self.changed_data or 'max_number_of_students' in self.changed_data) \
                 and len(students) > max_number_of_students:
             raise forms.ValidationError('Przekroczono limit przypisanych studentów.')
+        if "students" in self.data and self.cleaned_data['reserved_until'] is None:
+            raise forms.ValidationError("Do pracy przypisano studenta. Uzupełnij datę rezerwacji")
+        if "students" not in self.data and self.cleaned_data['reserved_until'] is not None:
+            raise forms.ValidationError("Nie przypisano studentów do pracy. Usuń datę rezerwacji")
```

Comment:
Zazwyczaj trzymaliśmy się umowy, że stringi „dla człowieka” są otoczone `"`, a te „dla komputera” `'`.

```suggestion
        if 'students' in self.data and self.cleaned_data['reserved_until'] is None:
            raise forms.ValidationError("Do pracy przypisano studenta. Uzupełnij datę rezerwacji")
        if 'students' not in self.data and self.cleaned_data['reserved_until'] is not None:
            raise forms.ValidationError("Nie przypisano studentów do pracy. Usuń datę rezerwacji")
```

Reply:
Ten wątek nie jest _resolved_, tj. zaproponowana zmiana realizująca ww. konwencję nie zostaje wdrożona.